go.work

```diff
@@ -1,7 +1,5 @@
-go 1.20
+go 1.21
 
-use (
-    ./src/app/authentication
-	
-	
```

Comment:
use go version 20 only.
don't make changes in this file

Reply:
installed the go version20